ios/browser/api/content_settings/default_host_content_settings.h

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_IOS_BROWSER_API_CONTENT_SETTINGS_DEFAULT_HOST_CONTENT_SETTINGS_H_
+#define BRAVE_IOS_BROWSER_API_CONTENT_SETTINGS_DEFAULT_HOST_CONTENT_SETTINGS_H_
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+// The mode in which pages should be loaded.
+typedef NS_ENUM(NSUInteger, DefaultPageMode) {
```

Comment:
this seems to be identical to the upstream file?

Reply:
I don't believe we can expose the upstream file since its not a public header so it'll have relative path includes and C++ types

Reply:
I don't see any other headers in default_page_mode.h

Reply:
Either way I don't want to use that directly, since this type–while duplicated–is explicitly for this Obj-C wrapper, and isn't actually used to interact with any of Chromium's APIs expecting that enum (simply using it for `CONTENT_SETTING_ALLOW`/`CONTENT_SETTING_BLOCK`